package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "uptime-kuma",
     "version": "1.22.1",
-    "lockfileVersion": 3,
+    "lockfileVersion": 2,
```

Comment:
The lock file is downgraded. Please use the latest npm version.

Reply:
My apologies, this has been corrected.